From 477624e087dccb88d2f870f6f271df195ffde1f5 Mon Sep 17 00:00:00 2001
From: Martijn Kaijser <mcm.kaijser@gmail.com>
Date: Mon, 28 Mar 2016 09:48:25 +0200
Subject: [PATCH] [AMLogic] Increase abs_error threshold to prevent video
 stutter on play/pause. As advised by Amlogic this value has to be set to
 0.300, instead of 0.150.

---
 xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp b/xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp
index 7236444..455ad94 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp
+++ b/xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp
@@ -1992,7 +1992,7 @@ void CAMLCodec::Process()
         if (abs_error > 0)
         {
           //CLog::Log(LOGDEBUG, "CAMLCodec::Process pts diff = %f", error);
-          if (abs_error > 0.150)
+          if (abs_error > 0.300)
           {
             // big error so try to reset pts_pcrscr
             SetVideoPtsSeconds(app_pts);
